samples/annotate/project-build.xml

```diff
@@ -159,10 +158,6 @@
 			<formatter type="xml"/>
 
 			<batchtest fork="yes" todir="${basedir}/target/junit-reports">
-				<fileset dir="${basedir}/src/test/java">
-					<include name="**/*Test*.java"/>
-					<exclude name="**/AllTests.java"/>
-				</fileset>
 				<fileset dir="${basedir}/target/generated-sources/xjc">
```

Comment:
The folders should exist. If they're not, it's a bug. The idea is to have an "just add water" project template. So the structure should have the directories. Maybe they get dropped as empty? In this case we should add some `.placeholder` or `README.md` file.